client/src/app/+my-account/my-account.component.ts

```diff
@@ -63,6 +65,21 @@ export class MyAccountComponent implements OnInit {
       })
     }
 
+    this.user = this.authService.getUser()
+
+    this.authService.userInformationLoaded.subscribe(
+      () => {
+        // Cannot see videos links, hide my videos and my imports
+        if (!this.canSeeVideosLink()) {
+          libraryEntries.children.splice(1, 1)
```

Comment:
A `isDisplayed (): boolean` method (default is `true`) in the `TopMenuDropdownParam` type would be more robust

Reply:
>  A isDisplayed (): boolean method (default is true) in the TopMenuDropdownParam type would be more robust

Good idea ! I sugguest optionnal isHidden () method, the default value will be undefined, so false in Angular template